packages/firestore/exp/test/deps/verify_dependencies.test.ts

```diff
@@ -1,50 +0,0 @@
-/**
```

Comment:
This is https://github.com/firebase/firebase-js-sdk/pull/3440. I cannot include this test since it uses `path`